docs/en/revocation-lists.rst

```diff
@@ -359,48 +382,62 @@ The *Credential status endpoint* MUST return a response with a *HTTP status code
-
-  - *error*. The error code.
-  - *error_description*. Text in human-readable form that offers more details to clarify the nature of the error encountered (for instance, changes in some attributes, reasons for revocation, other).
-
-Below a non-normative example of an HTTP Response with an error.
-
-.. code::
-
-  HTTP/1.1 400 Bad Request
-  Content-Type: application/json;charset=UTF-8
-
-  {
-    "error": "invalid_request"
-    "error_description": "The signature of credential_pop JWT is not valid"
-  }
-
-The following HTTP Status Codes and Error Codes MUST be supported:
+The following HTTP Status Codes MUST be supported:
 
 .. list-table:: 
     :widths: 20 20 60
     :header-rows: 1
 
     * - **Status Code**
-      - **Error Code**
+      - **Body**
       - **Description**
+    * - *201 Created*
+      - Status Attestation response
+      - The Status Attestation has been successfully created and it has been returned.  
     * - *400 Bad Request*
-      - *invalid_request*
-      - The request is not valid due to the lack or incorrectness of one or more parameters. (:rfc:`6749#section-5.2`).
+      - Error code and description
+      - The issuer cannot fulfill the request because of invalid parameters.
+    * - *404 Not Found*
+      - 
+      - The Digital Credential can not be found by the Issuer.
     * - *500 Internal Server Error*
-      - *server_error*
+      - 
       - The Issuer encountered an internal problem. (:rfc:`6749#section-5.2`).
     * - *503 Service Unavailable*
-      - *temporarily_unavailable*
+      - 
       - The Issuer is temporary unavailable. (:rfc:`6749#section-5.2`).
-    * - *400 Bad Request*
-      - *credential_revoked*
+
+For HTTP error responses that involve a body, the body MUST be encoded in ``application/json`` format and MUST contain the following parameters:
+
+  - *error*. The error code, as registerd in the table below.
+  - *error_description*. Text in human-readable form providing further details to clarify the nature of the error encountered.
+
+The following Error Codes MUST be supported:
+
+.. list-table:: 
+    :widths: 20 80
+    :header-rows: 1
+
+    * - **Error Code**
+      - **Description**
+    * - ``invalid_request``
+      - The request is not valid due to the lack or incorrectness of one or more parameters. (:rfc:`6749#section-5.2`).
+    * - ``credential_revoked``
       - The Digital Credential is revoked. The reason of revocation MUST be provided in the *error_description* field.
-    * - *400 Bad Request*
-      - *credential_updated*
+    * - ``credential_updated``
       - One or more attributes contained in the Digital Credential are changed. The *error_description* field MUST contain a list of updated attributes.
 
+Below a non-normative example of an HTTP Response with an error.
```

Comment:
duplicated: https://github.com/italia/eudi-wallet-it-docs/pull/221/files#diff-d3bfad4aa613161f55b96d232893081a0ac30d5127aca04dda54925b43aceb1eR250

Reply:
They are the same snippets but they apply to different endpoints.

I think it's ok to have examples when you expect them, even if they are duplicated